packages/manager/modules/pci/src/projects/project/storages/databases/database/logs/logs.controller.js

```diff
@@ -0,0 +1,81 @@
+import moment from 'moment';
+
+export default class logsCtrl {
+  /* @ngInject */
+  constructor($anchorScroll, $translate, $q, CucCloudMessage, DatabaseService) {
```

Comment:
unused injector: `$anchorScroll`, `$translate` and `$q`

Reply:
Removed with https://github.com/ovh/manager/commit/f63f3039f09d8eb53a20f7286e6f06666b538609